assets/app/view/game/game_info.rb

```diff
@@ -27,7 +27,21 @@ def render
       def render_body
         children = upcoming_trains
         children.concat(discarded_trains) if @depot.discarded.any?
-        children.concat(phases, game_info)
+        children.concat(phases)
+        children.concat(timeline_info) if timeline_info
```

Comment:
we could probably just call this timeline

Reply:
Ok, I will change that.